### PR TITLE
add arrow-poison tilecodes

### PR DESCRIPTION
**Pull request by estebanfer**
Adds tilecodes from https://github.com/spelunky-fyi/overlunky/pull/330 to the editor

### CODE REVIEW

**Comment by mauvealerts** [comment]
I *think* you can do the arrows by:
* add mappings in `items.py`  (see `metal_arrow`)
* add to the set in `tile_codes.py`

For `venom`, it's a little fuzzier since your sprite is rotated. If that's not important, you could add them similar to the arrows (see `poison_bubble1`). If you really want to re-orient it, I think the `tilecode_extras.py` and a custom sprite makes sense.

Sorry this took a while to review. I'm not very familiar with the level editor.

**Comment by estebanfer** [comment]
thanks for the advice, and no problem, it also took me a while to read this 😅

File: src/modlunky2/sprites/tilecode_extras.py [hunk 1]
```diff
@@ -127,9 +127,18 @@ class StickyTrapSheet(BaseSpriteLoader):
     }
 
 
+class PoisonSheet(BaseSpriteLoader):
+    _sprite_sheet_path = Path("static/images/arrows_poison.png")
```
Comment (mauvealerts) [inline src/modlunky2/sprites/tilecode_extras.py:131]: This needs to be updated to match the filename (`venom.png`)

File: src/modlunky2/sprites/items.py [hunk 2]
```diff
@@ -94,8 +95,11 @@ class ItemSheet(BaseSpriteLoader):
         "crossbow_empty_held": (2, 4, 3, 5),
         "crosbow_loaded_held": (3, 4, 4, 5),
         "metal_arrow": (4, 4, 5, 5),
+        "arrow_metal": (4, 4, 5, 5),
         "wood_arrow_poisoned": (5, 4, 6, 5),
         "metal_arrow_poisoned": (6, 4, 7, 5),
+        "arrow_wooden_poison": (5, 4, 6, 5),
+        "arrow_metal_poison": (6, 4, 7, 5),
```
Comment (mauvealerts) [inline src/modlunky2/sprites/items.py:102]: all of these (and `venom`) need to be added to `VALID_TILE_CODES` in `src\modlunky2\levels\tile_codes.py`

**Review by mauvealerts — COMMENTED** [review]
If you're having trouble testing, you might want to rebase. I fixed a bug in 20a3403be426bb808a19dff3a00d23c03e986952

File: src/modlunky2/sprites/items.py (hunk 2 above)
Comment (estebanfer) [reply src/modlunky2/sprites/items.py:102]: oops

Comment (estebanfer) [reply src/modlunky2/sprites/items.py:102]: done

File: src/modlunky2/sprites/tilecode_extras.py (hunk 1 above)
Comment (estebanfer) [reply src/modlunky2/sprites/tilecode_extras.py]: done

**Pull request merged**